### ISSUE TITLE
Add dynamic metatags to be read by seo4ajax bot

### ISSUE THREAD

**Issue by joshmorel** [FightPandemics/FightPandemics#1338]
Step towards resolving #909 

Likely will implement in same PR with #1337 

Notes: 
 
* We don't have post images yet so we will use the same one as the main site.
* Seems like 300 characters is good for description - https://stackoverflow.com/a/9552296 - we can take the first part of post body for now if longer

So for example https://staging.fightpandemics.work/post/5f04b73849aca117671ae4cd would have:

```html
<!-- DIFFERENT THAN INDEX (dynamic except og:type)  -->
<meta property="og:type" content="article" />

<meta name="title" content="COVID-19 Open Innovation Grants from Protocol Labs">
<meta property="og:title" content="COVID-19 Open Innovation Grants from Protocol Labs" />
<meta property="twitter:title" content="COVID-19 Open Innovation Grants from Protocol Labs" />

<meta name="description" content="The Protocol Labs Request for Proposals (RFP) increases our impact by supporting community pursuit of solutions to important problems. It works on the basis of discrete calls directed to specific open problems, for which we solicit short applications. Applications are welcome from anyone, regardl…" />
<meta property="og:description" content="The Protocol Labs Request for Proposals (RFP) increases our impact by supporting community pursuit of solutions to important problems. It works on the basis of discrete calls directed to specific open problems, for which we solicit short applications. Applications are welcome from anyone, regardl…" />
<meta property="twitter:description" content="The Protocol Labs Request for Proposals (RFP) increases our impact by supporting community pursuit of solutions to important problems. It works on the basis of discrete calls directed to specific open problems, for which we solicit short applications. Applications are welcome from anyone, regardl…" />

<meta property="og:url" content="https://staging.fightpandemics.work/post/5f04b73849aca117671ae4cd" />
<meta property="twitter:url" content="https://staging.fightpandemics.work/post/5f04b73849aca117671ae4cd" />


<!-- SAME AS INDEX FOR NOW -->
<meta name="image" content="https://staging.fightpandemics.work/Offer-Help-Request-Help.png" />
<meta property="og:image" content="https://staging.fightpandemics.work/Offer-Help-Request-Help.png" />
<meta property="twitter:image" content="https://staging.fightpandemics.work/Offer-Help-Request-Help.png" />
<meta property="twitter:card" content="summary_large_image" />
```

**Comment by joshmorel** [comment]
FYI  @manualzuru does the above seem correct?

Not sure about the non og/twitter items like `name="title"`/`name="description"`, not sure if this will be used by anything if the others exist but I guess it might not hurt?

**Comment by joshmorel** [comment]
Resolved by #1378

**Comment by anjanarao24** [comment]
This ticket is closed(Github prioritization exercise)
